test/functional/page_objects/discover_page.ts

```diff
@@ -133,7 +133,7 @@ export function DiscoverPageProvider({ getService, getPageObjects }: FtrProvider
 
       await browser
         .getActions()
-        .move({ x: 200, y: 20, origin: el._webElement })
+        .move({ x: 0, y: 20, origin: el._webElement })
```

Comment:
This change tested on both chromedriver and msedgedriver

Reply:
I had a failure on this test (Chrome headless) which I think might be affected by this change?
I'll do some additional testing on it.

```
         │ debg TestSubjects.click(superDatePickerAbsoluteTab)
         │ debg Find.clickByCssSelector('[data-test-subj="superDatePickerAbsoluteTab"]') with timeout=10000
         │ debg Find.findByCssSelector('[data-test-subj="superDatePickerAbsoluteTab"]') with timeout=10000
         │ debg TestSubjects.getAttribute(superDatePickerAbsoluteDateInput, value)
         │ debg TestSubjects.find(superDatePickerAbsoluteDateInput)
         │ debg Find.findByCssSelector('[data-test-subj="superDatePickerAbsoluteDateInput"]') with timeout=10000
         │ debg Find.findByCssSelector('tr.kbnDocTable__row:nth-child(1) > [data-test-subj='docTableField']') with timeout=10000
         │ info Taking screenshot "C:\Users\LeeDr\git\kibana\test\functional\screenshots\failure\discover app discover test query should modify the time range when the histogram is brushed.png"
         │ info Current URL is: http://localhost:5620/app/kibana#/discover/ddf4d550-7848-11ea-8dae-5932a1c158f2?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:%272015-09-19T20:44:44.254Z%27,to:%272015-09-20T22:07:57.566Z%27))
         │ info Saving page source to: C:\Users\LeeDr\git\kibana\test\functional\failure_debug\html\discover app discover test query should modify the time range when the histogram is brushed.html

  1) discover app
       discover test
         query
           should modify the time range when the histogram is brushed:
     Error: expected 1442804343667 to be within 1442804400000..1442809800000
      at Assertion.assert (packages\kbn-expect\expect.js:100:11)
      at Assertion.within (packages\kbn-expect\expect.js:265:8)
      at Function.within (packages\kbn-expect\expect.js:531:15)
      at Context.<anonymous> (test\functional\apps\discover\/_discover.js:103:43)
      at process._tickCallback (internal/process/next_tick.js:68:7)

         └- × fail: "discover app discover test query should modify the time range when the histogram is brushed"
```

Reply:
Chrome with head also failed with same result;
```
  1) discover app
       discover test
         query
           should modify the time range when the histogram is brushed:
     Error: expected 1442804343667 to be within 1442804400000..1442809800000
```


I checkout master and ran the same test\functional\apps\discover\_discover.js tests on Chrome headless and with head and all passed.

Reply:
Good catch, @LeeDr . The value you got is `Sep 20, 2015 @ 21:59:03.667` and it is just 1 min below the expected range `Sep 20, 2015 @ 22:00:00.000 - Sep 20, 2015 @ 23:30:00.000`.

My local results (Mac OS 10.14.6) are:
- Chrome 80 head: Sep 20, 2015 @ 22:17:57.757 
- Firefox 74 head: Sep 20, 2015 @ 22:17:57.757
- Chrome 80 headless: Sep 20, 2015 @ 22:17:57.757
- Firefox 74 headless:   Sep 20, 2015 @ 22:17:57.757

Jenkins (Linux):
- Chrome 80 headless: Sep 20, 2015 @ 22:17:57.757

So the value is constant across Unix-like OS in both head/headless.

with 0d79c58df2946c58f115aae2abb6a28a9fc1294a I shifted expected range to -30 min and it should work for Windows as well

Reply:
These tests all pass on the latest commit.